fabfile.py

```diff
@@ -89,7 +87,6 @@ def update():
         python_dependency(repo.name, PYTHON_VERSION)
 
     python_dependency('heltour', PYTHON_VERSION)
-    local("pip install -r {}".format(project_relative("requirements.txt")))
```

Comment:
I'm seeing two of these failures (see screenshot) - not sure if it's just this one or if one of the calls above it is also trying to install the old requirements.txt. Guess I could try standing up the vagrant again and see what it does

Reply:
![image](https://user-images.githubusercontent.com/3648900/153737191-0ba28e41-cc9b-4a27-9858-a1da61ef109c.png)

Looks good